app/src/vanilla/java/cm/aptoide/pt/app/view/AppViewFragment.java

```diff
@@ -613,15 +579,7 @@ public void buyApp(GetAppMeta.App app) {
     super.onCreateOptionsMenu(menu, inflater);
     this.menu = menu;
     inflater.inflate(R.menu.fragment_appview, menu);
-
-    final MenuItem menuItem = menu.findItem(R.id.menu_item_search);
-    if (appSearchSuggestionsView != null && menuItem != null) {
-      appSearchSuggestionsView.initialize(menuItem);
-    } else if (menuItem != null) {
-      menuItem.setVisible(false);
-    } else {
-      menu.removeItem(R.id.menu_item_search);
-    }
+    menu.removeItem(R.id.menu_item_search);
```

Comment:
Can't we just remove it from the xml instead?

Reply:
fixed